osu.Framework/Bindables/Bindable.cs

```diff
@@ -180,6 +180,17 @@ public IBindable<T> BindTarget
             set => ((IBindable<T>)this).BindTo(value);
         }
 
+        /// <summary>
+        /// Copies all values and value limitations of this bindable to another such.
+        /// </summary>
+        /// <param name="them">The target to copy to</param>
```

Comment:
Probably remove the "such"? Reads a bit unnatural...

```suggestion
        /// <summary>
        /// Copies all values and value limitations of this bindable to another.
        /// </summary>
        /// <param name="them">The target to copy to.</param>
```